v2/emailpassword/common-customizations/signup-form/changing-field-labels.mdx

```diff
@@ -1,49 +0,0 @@
----
-id: changing-field-labels
```

Comment:
set up redirection to the new file - search for `<Redirector` and see how it's used